config/defaults.go

```diff
@@ -37,4 +37,9 @@ func init() {
 	viper.SetDefault("database.name", "test")
 	viper.SetDefault("database.port", "5432")
 	viper.SetDefault("database.host", "localhost")
+
+	// Auth Defaults
+	viper.SetDefault("jwtKey", "GotFKl1PGgMpLg7D36NiI0hy/gsl6woTCXYdhKATbzc=")
```

Comment:
^^ I second this! Since this is an environment variable anyway, we can set it up in docker-compose or something. We should not have `jwt` token in code

Reply:
hm wasn't sure where to put it - do you think we should also move the db stuff out of there?

Reply:
yeah I think DB stuff should also not be in code! we should talk about this sunday meeting